qiime2/sdk/action.py

```diff
@@ -254,6 +257,8 @@ def _rewrite_wrapper_signature(self, wrapper):
     def _set_wrapper_properties(self, wrapper, name):
         wrapper.__name__ = wrapper.__qualname__ = name
         wrapper.__module__ = self.package
+        wrapper.__doc__ = "{}\n\n{}".format(
+            self.name, '\n'.join(textwrap.wrap(self.description, width=79)))
```

Comment:
This `\n` joining works great, but can you use [`textwrap.fill`](https://docs.python.org/3/library/textwrap.html#textwrap.fill) instead? That function will do the `wrap` and `\n` joining for you, so you can get this same functionality in one `textwrap.fill` call.